test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -448,6 +443,22 @@ public int size()
                                               i.config().localRack().equals(rackName));
     }
 
+    public void run(Consumer<I> action) {
```

Comment:
we already have `forEach`, rather than adding a new one, can we reuse the existing ones?

Reply:
removed this one